Gems/EMotionFX/Code/Include/Integration/ApplyMotionSetComponentBus.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#if defined(CARBONATED)
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Component/ComponentBus.h>
+#include <AzCore/Math/Vector2.h>
+#include <AzCore/Math/Vector3.h>
+#include <AzCore/Math/Quaternion.h>
+#include <AzCore/Asset/AssetCommon.h>
+#include <AzCore/Component/EntityId.h>
+
+namespace EMotionFX
+{
+    namespace Integration
+    {
+#if defined(CARBONATED)
+        enum MotionSetGender : int32_t
+        {
+            MOTION_NONE,
+            MOTION_NEUTRAL,
+            MOTION_FEMALE,
+            MOTION_MALE
+        };
+#endif
+
+        /**
+         * EmotionFX Anim Graph Component Request Bus
+         * Used for making requests to the EMotionFX Anim Graph Components.
+         */
+        class ApplyMotionSetComponentRequests
+            : public AZ::ComponentBus
+        {
+        public:
+
+            static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
+
+            // applies the defined motion set to this Entity's AnimGraphInstance
+#if defined(CARBONATED)

```

Comment:
Looks like inner #ifdef/#endif  for CARBONATED are not necessary due to we use #if defined(CARBONATED) for whole file in the line 3
?